### ISSUE TITLE
Cocktail model vs LoRA finetuning

### ISSUE THREAD

**Issue by huangtarn** [FlagOpen/FlagEmbedding#373]
Sorry for my ignorance. 
If we look at LoRA strategy, the weight change 
h=W_0*x+B*A*x
delta_W=B*A is the weight adjustment.
If we look at mono-special list model.
Mr<-alfpa*Mt+(1-alpha)*Mb.
Now the question is : Mr<-Mb+alpha*B*A
Fundamentally, Cocktail model is simply LoRA adjustment times a coefficient. How come it can have such a good performance?
Not sure if my understanding is correct.
thanks!

![image](https://github.com/FlagOpen/FlagEmbedding/assets/26346398/d4aa33fc-b52b-4292-9f10-6dbbbb56eb94)

![image](https://github.com/FlagOpen/FlagEmbedding/assets/26346398/815a1429-7bc9-4751-b19e-733228ebca5a)

**Comment by staoxiao** [comment]
The LoRA technique adds the delta parameter during training, optimizing the loss by calculating it based on the original model parameters augmented with the delta parameter. This approach helps prevent overfitting but still carries the risk of catastrophic forgetting. In contrast, our method optimizes the delta model separately, and merges models after the training process.
